README.md

```diff
@@ -1 +1,122 @@
-# Hello-Microverse
\ No newline at end of file
+<a name="readme-top">SET UP LINTERS AND CREATE PULL REQUEST</a>
+
+
+<div align="center">
+
+  <img src="images/images.png" alt="logo" width="140"  height="auto" />
+  <br/>
```

Comment:
- [ ] As requested by the previous reviewer, you need to delete this img tag because it's displaying correctly.